src/mumble/Database.h

```diff
@@ -54,8 +55,9 @@ class Database : public QObject {
 	QString getUserLocalNickname(const QString &hash);
 	void setUserLocalNickname(const QString &hash, const QString &nickname);
 
-	bool isChannelFiltered(const QByteArray &server_cert_digest, const int channel_id);
-	void setChannelFiltered(const QByteArray &server_cert_digest, const int channel_id, bool hidden);
+	Channel::FilterMode getChannelFilterMode(const QByteArray &server_cert_digest, const int channel_id);
+	void setChannelFilterMode(const QByteArray &server_cert_digest, const int channel_id,
+							  Channel::FilterMode filterMode);
```

Comment:
```suggestion
	Channel::FilterMode getChannelFilterMode(const QByteArray &server_cert_digest, int channel_id);
	void setChannelFilterMode(const QByteArray &server_cert_digest, int channel_id,
							  Channel::FilterMode filterMode);
```
Since we're changing these lines anyway, we can also remove the `const` which does not have any effect on a value-type in a function's _declaration_.